### PR TITLE
fix(language-selector): update language selector to use select

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#4330, #4331 

### Description

This change adds Carbon's `Select` component for the footer language selector in mobile/tablet breakpoints. Desktop views will continue to use `ComboBox`.

### Changelog

**Changed**

- `Footer` language selector now uses Carbon's `Select` instead of `ComboBox` for mobile/tablet breakpoints
- updated various styles for new component

### CODE REVIEW

File: packages/react/src/components/Footer/LanguageSelector.js [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
  */
 
-import React, { useState, useEffect, useRef } from 'react';
-import ComboBox from '../../internal/vendor/carbon-components-react/components/ComboBox/ComboBox';
+import React, { useState } from 'react';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
 import PropTypes from 'prop-types';
-import root from 'window-or-global';
+import Select from '../../internal/vendor/carbon-components-react/components/Select/Select';
+import SelectItem from '../../internal/vendor/carbon-components-react/components/SelectItem/SelectItem';
```
Comment (jeffchew) [inline packages/react/src/components/Footer/LanguageSelector.js:12]: I thought the idea was that we would only use select for mobile/tablet, and the combobox remains for desktop?

Comment (kennylam) [reply packages/react/src/components/Footer/LanguageSelector.js:12]: @jeffchew Yep, you are right. Confirmed with @wonilsuhibm that it should use combobox for desktop.

**Comment by Praveen-sr** [comment]
@kennylam Re tested on the new PR link, above issue is fixed, Working fine now.

But new issue is found, 
Issue: There is no label for this dropdown

Expected: Provide label for the dropdown using html label tag or aria label

**Review by jeffchew — APPROVED** [review]
LGTM!

Looks like Praveen has some feedback.

**Review by RaphaelAmadeu — APPROVED** [review]
lgtm!

**Pull request merged**